nucypher/cli/actions/configure.py

```diff
@@ -50,6 +50,23 @@ def forget(emitter: StdoutEmitter, configuration: CharacterConfiguration) -> Non
     emitter.message(SUCCESSFUL_FORGET_NODES, color='red')
 
 
+def get_config_filepaths(config_class: Type[CharacterConfiguration], config_root: str = None) -> List:
+    #
+    # Scrape disk for configuration files
+    #
+    config_root = config_root or DEFAULT_CONFIG_ROOT
+    default_config_file = glob.glob(config_class.default_filepath(config_root=config_root))
+
+    # updated glob pattern for secondary configuration files accommodates for:
+    # 1. configuration files with "0x..." checksum address as suffix - including older ursula config files
+    # 2. newer (ursula) configuration files which use signing_pub_key[:8] as hex as the suffix
+    glob_pattern = f'{config_root}/{config_class.NAME}-[0-9a-fA-f]*.{config_class._CONFIG_FILE_EXTENSION}'
```

Comment:
Shouldn't it be `0-9a-fA-F` (the last F capital)?

Reply:
Also feels like we should be matching for the exact number of symbols here (8), but `glob` can't do it by itself, so it will require additional filtering. I wonder what will happen if there's a file with less or more than 8 symbols satisfying the pattern.

Reply:
> Shouldn't it be 0-9a-fA-F (the last F capital)?
Yes - thanks! ✔️ 

It's just a question of backwards compatibility. The glob needs to match both
- <config_class>-<worker_address>.json (backwards compatibility)
- <config_class>-<signing_public_key[:8] (new naming)

I don't think random files following the naming pattern would be something we need to worry about.

Reply:
Technically we could match these two specific variants (with postprocessing of the glob results), but I agree that this is a minor thing and probably won't cause problems.